packages/Notification/__tests__/Notification.spec.jsx

```diff
@@ -44,12 +43,6 @@ describe('<Notification />', () => {
   })
 
   describe('successful variant', () => {
-    it('bolds the content', () => {
```

Comment:
Do we want to test for default `font-weight`?

Reply:
Done